app/helpers/custom_links_helper.rb

```diff
@@ -6,9 +6,9 @@ def custom_links
     begin
       return result if source.nil?
       json = JSON.parse(source, quirks_mode: true)
-      json.each do |j|
-        if j.is_a?(Hash) && j.key?('label') && j.key?('url')
-          result += "<li>#{link_to j['label'], j['url']}</li>"
+      json.each do |elm|
+        if elm.is_a?(Hash) && elm.key?('label') && elm.key?('url')
+          result += "<li>#{link_to elm['label'], elm['url']}</li>"
```

Comment:
__[Reek]__

[FeatureEnvy] CustomLinksHelper#custom_links refers to 'elm' more than self (maybe move it to another class?)


<!-- Comment by [SideCI](https://sideci.com) -->